lib/broadside/command.rb

```diff
@@ -34,22 +34,6 @@ def targets(options)
         info "\n" + table.render(:ascii, padding: [0, 1])
       end
 
-      def deploy_short(options)
```

Comment:
the point of this module and repeating all this was to create an abstraction that would separate GLI command config from actual execution logic